src/PermissionsContext.tsx

```diff
@@ -34,7 +35,9 @@ const toENStatus = (enPermissionStatus: ENPermissionStatus): ENStatus => {
   const isEnabled = enPermissionStatus[1] === "ENABLED"
 
   if (!isAuthorized && !isEnabled) {
-    return ENStatus.UNAUTHORIZED_DISABLED
+    return isPlatformiOS()
```

Comment:
@johnschoeman @aledustet What are your thoughts on this? This means on Android the possible states for `exposureNotifications.status` are `ENStatus.AUTHORIZED_DISABLED` and `ENStatus.AUTHORIZED_ENABLED`. So we only expose an unauthorized state from the provider on iOS.

Reply:
That logically makes sense to me, but i find the naming a bit confusing or at least the motivation behind this is hard to glean from reading the code.  So I wonder if the ENStatus enum values are too iOS focused?

Reply:
I agree @johnschoeman, I think https://github.com/Path-Check/gaen-mobile/pull/334/commits/b746bfa292d769abd38e68c90bbc5cc53cfe631a is a step in the right direction, but I'm confused as to why we can end up in a state where exposure notification permissions are unauthorized on Android. Shouldn't we always be authorized on Android, in which case we only care if we are enabled or disabled?